frontend/src/styles/theme.jsx

```diff
@@ -0,0 +1,44 @@
+import { createTheme } from '@mui/material/styles';
+import { blueGrey } from '@mui/material/colors';
+import '@fontsource/poppins';
+
+const theme = createTheme({
+    spacing: 8,
+    palette: {
+        primary: {
+            main: blueGrey[500],
+        },
+        secondary: {
+            main: '#fff',
```

Comment:
Avoid using hard code to define colors, making it harder to maintain consistency between components in the future. Instead, import from the library like you did in line 9 "blueGrey[500]" or define constants like `const colorX = '#some hexadecimal code'`. Let's take the library and import the colors from there.